### PR TITLE
feat(scion-rcp): extract scion microfrontend API interfaces

### PR DESCRIPTION
**Pull request by ruadrian**
* Faciliates decorating these services.

### CODE REVIEW

**Comment by ruadrian** [comment]
Wait for feedback

File: ch.sbb.scion.rcp.microfrontend/src/ch/sbb/scion/rcp/microfrontend/browser/RxJsObservable.java
Comment (ruadrian) [inline ch.sbb.scion.rcp.microfrontend/src/ch/sbb/scion/rcp/microfrontend/browser/RxJsObservable.java:1]: Undo these unrelated changes

**Review by karacsonybarni — APPROVED** [review]
